mirage/utils/siaf_interface.py

```diff
@@ -37,7 +38,16 @@ def get_siaf_information(instrument, aperture, ra, dec, telescope_roll):
     aperture : str
         Aperture name (e.g. "NRCA1_FULL")
     """
-    siaf = pysiaf.Siaf(instrument)[aperture]
+    # Temporary fix to access good NIRCam distortion coefficients which
+    # which are not yet in the PRD
+    if instrument.lower() == 'nircam':
+        import os
+        print("NOTE: Using pre-delivery SIAF data")
+        pre_delivery_dir = os.path.join(JWST_DELIVERY_DATA_ROOT, instrument)
+        siaf = pysiaf.Siaf(instrument, basepath=pre_delivery_dir)[aperture]
```

Comment:
This will fail when the updated NIRCam SIAF will have been delivered to PRD and the pre_delivery_dir will be empty. I think this is fine, because it will make sure that this code is removed when it is no longer needed.